packages/fx-core/src/common/tools.ts

```diff
@@ -451,3 +451,37 @@ export function getAppStudioEndpoint(): string {
     return "https://dev.teams.microsoft.com";
   }
 }
+
+export async function copyFiles(
```

Comment:
Is it possible that we just use some popular library? like https://www.npmjs.com/package/recursive-copy